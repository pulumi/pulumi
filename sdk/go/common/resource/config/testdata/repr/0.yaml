my:anotherTestKey: anotherTestValue
my:testKey: testValue
