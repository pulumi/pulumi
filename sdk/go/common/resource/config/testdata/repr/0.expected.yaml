decrypt:
  my:anotherTestKey: anotherTestValue
  my:testKey: testValue
paths:
  my:anotherTestKey:
    value: anotherTestValue
    string: anotherTestValue
    redacted: anotherTestValue
    object: HRAABnN0cmluZwwSABBhbm90aGVyVGVzdFZhbHVl
    secure: false
    isObject: false
  my:testKey:
    value: testValue
    string: testValue
    redacted: testValue
    object: FhAABnN0cmluZwwLAAl0ZXN0VmFsdWU=
    secure: false
    isObject: false
